base/common/src/main/java/org/dogtagpki/nss/NSSExtensionGenerator.java

```diff
@@ -197,20 +200,35 @@ public SubjectKeyIdentifierExtension createSKIDExtension(X509Key subjectKey) thr
 
         logger.info("Creating SKID extension:");
 
-        byte[] bytes;
+        boolean critical = false;
+        byte[] bytes = null;
+
+        List<String> values = Arrays.asList(subjectKeyIdentifier.split(","));
+        for (String value : values) {
+            value = value.trim();
+            logger.info("- " + value);
+
+            if (value.equals("critical")) {
+                critical = true;
+                continue;
+            }
 
-        if (subjectKeyIdentifier.equals("hash")) {
-            logger.info("- hash");
-            bytes = CryptoUtil.generateKeyIdentifier(subjectKey.getKey());
+            if (value.equals("hash")) {
+                bytes = CryptoUtil.generateKeyIdentifier(subjectKey.getKey());
+                continue;
+            }
 
-        } else {
-            throw new Exception("Unsupported subjectKeyIdentifier: " + subjectKeyIdentifier);
+            try {
+                bytes = HEX_FORMAT.parseHex(value);
```

Comment:
If called by the python code is OK but if called by other client we should support the value `none`.

Reply:
I think the `none` actually could be confusing. Let's say we have this:
```
subjectKeyIdentifier = critical, none
```
Should we not create the extension at all? Or should we create a critical one but with an empty identifier? I'm not sure if that's even allowed.

I think if we don't want the extension we should just not specify `subjectKeyIdentifier` at all in the configuration.